operating/manage-user-access/authentication-provider-structure.adoc

```diff
@@ -0,0 +1,86 @@
+:_content-type: ASSEMBLY
+[id="authentication-provider-structure"]
+= Authentication provider structure
+include::modules/common-attributes.adoc[]
+:context: authentication-provider-structure
+
+toc::[]
+
+[role="_abstract"]
+
+An authentication provider connects to a third-party source of user identity (for example, an identity provider, or IdP). An authentication provider gets the user identity from IdP, issues a token based on this identity, and returns the token to the caller. The mentioned token is then sent within the user interface or API calls and allows {product-title-short} to authorize the user.
+
+You can set up your own IdP after provisioning by navigating to *Operating* -> *Managing user access* and selecting the appropriate module. 
```

Comment:
This sentence is in passive voice. 

s/ Navigate to `Operating -> Managing user access`, and select the appropriate module to set up your own IDP immediately after provisioning.